src/wordy_webserver_response.js

```diff
@@ -21,6 +23,10 @@ function userOut(storage, userId, callback) {
   });
 }
 
+function ping(storage, userId, callback) {
```

Comment:
Ha! I knew you would notice.

Ummmm, unless you can think of a **simple** way of improving the implementation, I'd go ahead with it.